cypress/integration/collections_spec.js

```diff
@@ -0,0 +1,83 @@
+describe('Dashboard Collections Page', () => {
+  const host = process.env.DASHBOARD_HOST || 'http://localhost:3000/';
+
+  describe('When not logged in', () => {
+    it('should redirect to login page', () => {
+      cy.visit(`${host}#/collections`);
+      cy.url().should('include', '/#/auth');
```

Comment:
I create a [re-usable assertion](https://github.com/nasa/cumulus-dashboard/pull/497/files#diff-bbe16e111202074abf127c887faa6d32R1) for this in #497 to make it easier to re-test this

Reply:
yes.  I will make updates after you merge.